docs/security.md

```diff
@@ -53,3 +53,13 @@ For further information check the links down below:
 * https://www.owasp.org/index.php/Main_Page
 * https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#javascript-guidance-for-auto-inclusion-of-csrf-tokens-as-an-ajax-request-header
 * https://www.owasp.org/index.php/Category:OWASP_Top_Ten_Project
+
+### Sign in brute force protection
+
+To protect those users who don't have OTP enabled from brute-force login attacks we use `rack-attack` gem.
```

Comment:
for now we have this check even for users with otp, as we are using same endpoint?

Reply:
Yes, we have it for all users, I've added this part just to make it clear that users with otp protected against brute force by otp itself.